package.json

```diff
@@ -12,26 +12,26 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^16.2.10",
+    "@angular/animations": "^18.2.7",
     "@angular/cdk": "^16.2.9",
```

Comment:
Do not forget to update angular material as well...

Reply:
created a pr for this issue